### ISSUE TITLE
10.0.22621.3640

### ISSUE THREAD

**Issue by RUNG5445** [stascorp/rdpwrap#2620]
x64

**Comment by AllenTan7389** [comment]
> x64

can u provide termsrv.dll ?

**Comment by RUNG5445** [comment]
> > x64
> 
> can u provide termsrv.dll ?

Sorry , Here is termsrv.dll @AllenTan7389 
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/15413182/termsrv.zip)

**Comment by AllenTan7389** [comment]
> > > x64
> > 
> > 
> > can u provide termsrv.dll ?
> 
> Sorry , Here is termsrv.dll @AllenTan7389 [termsrv.zip](https://github.com/stascorp/rdpwrap/files/15413182/termsrv.zip)

Done , please replace this termsrv.dll to solve your RDP issue 
before replace please unistall your RDP Wrap 
here the link https://we.tl/t-yn81yCi4Tr

**Comment by RUNG5445** [comment]
> > > > x64
> > > 
> > > 
> > > can u provide termsrv.dll ?
> > 
> > 
> > Sorry , Here is termsrv.dll @AllenTan7389 [termsrv.zip](https://github.com/stascorp/rdpwrap/files/15413182/termsrv.zip)
> 
> Done , please replace this termsrv.dll to solve your RDP issue before replace please unistall your RDP Wrap here the link https://we.tl/t-yn81yCi4Tr

Is it safe?

**Comment by AllenTan7389** [comment]
yes that termsrv.dll have been unlocked so no need use any rdp wrap ...

**Comment by binarymaster** [comment]
@RUNG5445 do not replace any system files, this is not allowed by the project's policy. Offending user has been banned.

If you replaced the file, run `sfc /scannow` and check your system for viruses.

**Comment by loyejaotdiqr47123** [comment]
[https://github.com/sebaxakerhtc/rdpwrap.ini/commit/80af777e0f630e1a52f6c544a95151d58450f21c](https://github.com/sebaxakerhtc/rdpwrap.ini/commit/80af777e0f630e1a52f6c544a95151d58450f21c)

**Comment by loyejaotdiqr47123** [comment]
@binarymaster Please reopen
